.gitignore

```diff
@@ -54,7 +58,7 @@ playground.xcworkspace
 # you should judge for yourself, the pros and cons are mentioned at:
 # https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 #
-# Pods/
+Pods/Mapbox-iOS-SDK
```

Comment:
Please also delete the existing pods directory so that all the files don't show up in the PR.